qa/automation/CareerPortal.Tests.Scenarios/samsonova_sendCV/send_cv.feature

```diff
@@ -0,0 +1,40 @@
+  Feature: Send CV
+
+  Scenario: positive sending CV
+  Given form "Send CV" with blank fields
+  When I enter valid data in the field "Name"
+  And I enter valid data in the field "Email"
+  And I attach CV
+  And I ticked "I agree to the personal data processing"
+  And I click button "Send"
+  Then my CV has been send
```

Comment:
Perhaps we also can write some more result that we can see? For example: error message or indicator of some colour.

Reply:
done